SpatialGDK/Source/SpatialGDK/Public/Utils/SpatialStatics.h

```diff
@@ -0,0 +1,36 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#pragma once
+
+#include "Kismet/BlueprintFunctionLibrary.h"
+#include "SpatialStatics.generated.h"
+
+DECLARE_LOG_CATEGORY_EXTERN(LogSpatialStatics, Log, All);
+
+UCLASS()
+class SPATIALGDK_API USpatialStatics : public UBlueprintFunctionLibrary
+{
+	GENERATED_BODY()
+
+public:
+	/**
+	 * Returns whether Spatial networking has been enabled or not */
+	UFUNCTION(BlueprintPure, Category = "Game")
```

Comment:
I think the Category should be `SpatialOS` or at least match the same category as the SpatialFlags class